app/src/main/kotlin/net/corda/cli/application/logger/LogToFileAppender.kt

```diff
@@ -0,0 +1,35 @@
+package net.corda.cli.application.logger
+
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.core.appender.FileAppender
+import org.apache.logging.log4j.core.layout.PatternLayout
+import java.io.File
+
+
+class LogToFileAppender {
+    companion object {
+        fun enableLogToFile(filePath: File, level: String) {
```

Comment:
I think we are somewhat duplicating what we have in `log4j2.xml`.
If we need to control log level and location of the file, I think we should be able to do that directly in `log4j2.xml` like so:
```
<Root level="${env:LOG_LEVEL:-info}">
```
i.e. reading system properties directly.

Also, when we are adding an `FileAppender` here programmatically, are we sure we are not going to end-up with **3** appenders in this case:
1. Stdout;
2. Rolling file appender from `log4j2.xml`; 
3. `logToFileAppender` configured here
?

My preference would be to get rid of this class and have all the logging configuration settings in `log4j2.xml`.